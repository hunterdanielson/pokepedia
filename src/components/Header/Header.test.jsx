import React from 'react';
import { shallow } from 'enzyme';
import Header from './Header';
import AppProvider from '../../providers/appProvider';

describe('header component', () => {
  it('renders Header', () => {
    const wrapper = shallow(<AppProvider><Header /></AppProvider>);
    expect(wrapper).toMatchSnapshot();
  });
});
  
