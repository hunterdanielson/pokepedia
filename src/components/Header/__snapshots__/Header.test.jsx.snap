// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`header component renders Header 1`] = `
<ContextProvider
  value={
    Object {
      "dispatch": [Function],
      "state": Object {
        "list": Array [],
        "pageNum": 1,
        "theme": "light",
        "totalCount": 964,
      },
    }
  }
>
  <button
    name="next"
    onClick={[Function]}
  >
    Next
  </button>
  <Header />
</ContextProvider>
`;
